docs/UserGuide.md

```diff
@@ -526,7 +529,7 @@ Adds a new item to the list of todo items.
 
 Format: `delete-e INDEX`
 
-* Deletes the event at the specified INDEX. The index refers to the index number shown in the displayed event list (list-e). The index must be a **positive integer** (1, 2, 3, …).
+- Deletes the event at the specified INDEX. The index refers to the index number shown in the displayed event list (list-e). The index must be a **positive integer** (1, 2, 3, …).
```

Comment:
Why was this change made ah? I don't think there is a difference to the output right?

Reply:
Just for standardization since all are using '-'